templates/trivy/trivy-sts.yaml

```diff
@@ -93,6 +93,8 @@ spec:
               value: {{ .Values.trivy.ignoreUnfixed | default false | quote }}
             - name: "SCANNER_TRIVY_SKIP_UPDATE"
               value: {{ .Values.trivy.skipUpdate | default false | quote }}
+            - name: "SCANNER_TRIVY_SKIP_DB_UPDATE"
```

Comment:
Hi @AlawnWong, thanks for you contribute to harbor-helm.

I don't think we need to set `SCANNER_TRIVY_SKIP_DB_UPDATE` to `true` as well. Reason:
1. there is no env called `SCANNER_TRIVY_SKIP_DB_UPDATE` in https://github.com/aquasecurity/harbor-scanner-trivy
2. the existing `SCANNER_TRIVY_SKIP_UPDATE` is used to call `--skip-db-update`
* https://github.com/aquasecurity/harbor-scanner-trivy/blob/main/pkg/etc/config.go#L35
* https://github.com/aquasecurity/harbor-scanner-trivy/blob/main/pkg/trivy/wrapper.go#L136